test/example.test.js

```diff
@@ -0,0 +1,43 @@
+// IMPORT MODULES under test here:
+// import example from '../src/example.js';
+
+const test = QUnit.test;
+
+test('time to test a function', function(assert) {
```

Comment:
You should comment out or delete this sample test. It makes it appear as if all your tests don't pass in the browser